wasm/tx_transfer/src/lib.rs

```diff
@@ -53,7 +53,9 @@ fn apply_tx(ctx: &mut Ctx, tx_data: BatchedTx) -> TxResult {
             .wrap_err("Encountered error while handling MASP transaction")?;
         update_masp_note_commitment_tree(&shielded)
             .wrap_err("Failed to update the MASP commitment tree")?;
-        ctx.push_action(Action::Masp(MaspAction { masp_section_ref }))?;
+        ctx.push_action(Action::Masp(MaspAction::MaspSectionRef(
```

Comment:
The MASP integration tests seem to be failing for me. Probably they are failing on at least the shielding transactions because their signers have not been added as actions. So I think we need to push some `MaspSigner` `Action`s in this vicinity - the same set that will be iterated through in the `for signer in signers {` loop if we desire to support the Christel thieving corner case above. But at the least, all of the net debited accounts should at least be added to the `MaspSigner` `Action`s. No?

Reply:
That's correct. I'm fixing this issue rn